private/buf/bufworkspace/workspace_targeting.go

```diff
@@ -783,3 +840,31 @@ func checkForOverlap(
 	}
 	return fmt.Errorf("input %q did not contain modules found in workspace %v", inputPath, moduleDirPaths)
 }
+
+func formatBucketIDForDuplicateModulePath(moduleDirPath string, nthDuplicate int) string {
```

Comment:
Why are we doing all this pretty-print work for bucket IDs?

Reply:
With the PR adding `description()`, this no longer needs to be pretty-printed, but before that PR this is meant to be surfaced in error messages.

No longer pretty-formatting, fixed in [499a77b](https://github.com/bufbuild/buf/pull/3260/commits/499a77b62faf79786243dd77a25085155bb57164)